src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -52,12 +93,19 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		statsd.increment(metricCreateBranch.created, tags);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.status(err.status).json(errorMessages[err?.status || 500]);
+
+		if (err.status === 403) {
+			const user = await gitHubUserClient.getUser();
+			const gitHubConfigurationLink = await getGitHubConfigurationLink(user.data.login, jiraHost, gitHubAppConfig.hostname, owner, repo);
+			res.status(err.status).json(errorMessages(err?.status || 500, gitHubConfigurationLink));
+
+		} else {
+			res.status(err.status).json(errorMessages(err?.status || 500));
```

Comment:
`err.status` might be not defined here, right? How would `res.status(undefined)` behave?

Reply:
Updated to `err.status || 500`. If its undefined, it will take the default 500.